src/Actions/MenuEditor.php

```diff
@@ -200,7 +200,7 @@ public function add_editor_menu_capabilities() {
 
         $current_user = wp_get_current_user();
 
-        if ( current_user_can( 'editor' ) && ( $current_user->caps['editor'] ) ) {
+        if ( current_user_can( 'editor' ) && ( isset( $current_user->caps['editor'] ) ) ) {
```

Comment:
```suggestion
        if ( current_user_can( 'editor' ) && ( $current_user->has_cap( 'editor' ) ) ) {
```
Are we able to use the WP_User->has_cap() (https://developer.wordpress.org/reference/classes/WP_User/has_cap/) function here?

Reply:
This one was giving me some problems locally (story of my life). I wouldn't mind digging into it a bit more, but for now checking the capabilities index seems to do the trick.